dev_docs/cxg.md

```diff
@@ -0,0 +1,162 @@
+
+CXG supports zero or more embeddings. Note that cellxgene currently _requires_ at least one embedding.
+
+### cxg_group_metadata
+
+Required, but empty TileDB array, used to store CXG-wide metadata.  The following fields are defined:
+* __cxg_version__: (required) a semver string identifying the specification version used to encode the CXG.
+* __cxg_properties__: (optional) a dictionary containing dataset wide properties, defined below.
+* __cxg_category_colors__: (optional) a categorical color table, defined below.
+
+#### cxg_properties
+
+The properties metadata dictionary contains dataset-wide properties, encoded as a JSON dictionary. Curently, the following fields are defined:
+* title: string, dataset human name (eg, "Lung Tissue")
+* about: string, fully-qualified http/https URL, linking to more inforation on the dataset.
+
+All implementions MUST ignore unrecognized fields. 
+
+#### cxg_category_colors
+
+This optional field contains a copy of the category color table, which MAY be used to display category-specific color labels. This is a JSON dictionary, containing a per-category color-table.  Each color table is named `{category_name}_colors`, and is itself a dictionary mapping label name to RGB color.  For example:
+
+```
+{
+    "louvain_colors": {
+        "0": "#FFFFFF",
+        "1": "#000000"
+    }
+}
+```
+
+## Corpora Schema Encoding
+
+The [Corpora schema](https://github.com/chanzuckerberg/corpora-data-portal/blob/main/backend/schema/corpora_schema.md) and [Corpora AnnData encoding](https://github.com/chanzuckerberg/corpora-data-portal/blob/main/backend/schema/corpora_schema_h5ad_implementation.md) define a set of metadata and encoding conventions for annotated matrices.  When a Corpora dataset is encoded as a CXG, the following shall apply.
+
+### Corpora metadata property
+
+A CXG containing a Corpora dataset will contain a property in the __cxg_group_metadata__ field named `corpora`. The value will be a JSON encoded string, which in turn contains all properties defined in the [Corpora AnnData uns](https://github.com/chanzuckerberg/corpora-data-portal/blob/main/backend/schema/corpora_schema_h5ad_implementation.md#uns) container. For example:
+
+```
+{
+    "corpora": {
+        "version": {
+            "corpora_schema_version": "1.0.0",
+            "corpora_encoding_version": "0.1.0",
+        }
+    }
+}
+```
+
+The `corpora` metadata, if present, MUST contain the version information. Optionality of other values in this object will follow the specifications set forth in the relevant Corpora schema specification (ie, optional fields are optional, required are present, etc), with the following changes:
+* the contents of `corpora_encoding_version` MUST be identical to the `cxg_version`, as this field is defined as the current object encoding version, *NOT* the source data encoding version.
```

Comment:
If corpora_encoding_version must be identical to cxg_version, then why have it?

Reply:
Because cxg_version is a super set.   They evolve independently.